trendmicro_deepsecurity/icon_trendmicro_deepsecurity/actions/deploy_rules/action.py

```diff
@@ -0,0 +1,77 @@
+import komand
+from .schema import DeployRulesInput, DeployRulesOutput, Input, Output, Component
+# Custom imports below
+
+import requests
+import json
+
+from icon_trendmicro_deepsecurity.util.shared import tryJSON
+from icon_trendmicro_deepsecurity.util.shared import checkResponse
+
+class DeployRules(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='deploy_rules',
+                description=Component.DESCRIPTION,
+                input=DeployRulesInput(),
+                output=DeployRulesOutput())
+
+    def run(self, params={}):
+        """
+        Set IPS rules in Deep Security
+        """
+
+        # Get parameters
+        self.computer_or_policy=params.get(Input.COMPUTER_OR_POLICY)
```

Comment:
Use spaces between `=` for style conformance

Reply:
done